### DISCUSSION TITLE
SB5.1 - OCI - Deploy/UCI/Inbound - Values information

### DISCUSSION THREAD

**Discussion by ddevadat** [Sunbird-Ed/Community#380] [Installation] [answered]
We are deploying Deploy/UCI/Inbound on oci for release 5.1.0

The deployment is failing with the below error

```
TASK [sunbird-deploy : copy and template env file from swarm role] *************
failed: [localhost] (item=../../../../ansible/roles/stack-sunbird/templates/inbound.env) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": "../../../../ansible/roles/stack-sunbird/templates/inbound.env"
}

MSG:

AnsibleUndefinedVariable: 'uci_odk_user' is undefined
```

Could you please let us know what should be the values for 
uci_odk_user
uci_odk_password
fusionauth_service_admin_key
uci_api_admin_token
uci_netcore_whatsapp_token
uci_netcore_whatsapp_source
uci_netcore_whatsapp_uri

**Comment by beepdot** [comment]
@saiakhil46 Please check this thread and raise a PR for private repo templates.

**Comment by ddevadat** [comment, marked as answer]
any update on this

**Reply by Krishnaj20** [reply]
@gandham-santhosh - Pls respond.

**Reply by saiakhil46** [reply]
@ddevadat  we are adding these details in our template, we will share you the details shortly.

**Reply by saiakhil46** [reply]
@ddevadat This was answered in https://github.com/Sunbird-Ed/Community/discussions/373 discussion please refer to it
